api/src/test/java/uk/gov/dhsc/htbhf/claimant/ClaimantServiceIntegrationTestsWithScheduledServices.java

```diff
@@ -68,6 +70,25 @@ void shouldRequestNewCardAndSendEmailForSuccessfulClaim() throws JsonProcessingE
         wiremockManager.assertThatDepositFundsRequestMadeForPayment(payment);
     }
 
+    @Test
+    void shouldSendWeWillLetYouKnowEmailForAddressMismatch() throws JsonProcessingException, NotificationClientException {
+        ClaimDTO claimDTO = aValidClaimDTOWithNoNullFields();
+        wiremockManager.stubEligibilityResponse(anIdMatchedEligibilityConfirmedAddressNotMatchedResponse());
+        stubNotificationEmailResponse();
+
+        ResponseEntity<ClaimResultDTO> response = restTemplate.exchange(buildClaimRequestEntity(claimDTO), ClaimResultDTO.class);
+        invokeAllSchedulers();
+
+        assertThat(response.getStatusCode()).isEqualTo(OK);
+        assertThat(response.getBody().getClaimStatus()).isEqualTo(ClaimStatus.REJECTED);
+        Claim claim = repositoryMediator.getClaimForNino(claimDTO.getClaimant().getNino());
+        assertThat(claim.getClaimStatus()).isEqualTo(ClaimStatus.REJECTED);
```

Comment:
Do we need to check the claim status both on the response and in the database?

Reply:
We do that in other tests too. I guess we don't need to here as well.